Fix nits in Modular Resources (#1435)
viamrobotics/docs: docs/extend/modular-resources/_index.md

{{% alert title="Modules vs. modular resources" color="tip" %}}

A configured *module* can make one or more *modular resources* available for configuration.

{{% /alert %}}

### Code your module

> Use [Viam's Go or Python language SDKs](/program/apis/) to code a module that does either of the following:

Author: Can you actually do both in one module? If so
```suggestion
Use [Viam's Go or Python language SDKs](/program/apis/) to code a module that does either or both of the following:
```

Reviewer: I think as is makes sense. You'll probably want to do either or most of the time?

Reviewer: Maybe remove the word language? Feels a bit strange

Author: Okay! I was attempting to preserve the original sentence information but I agree it sounds strange :)
```suggestion
Use [Viam's Go or Python SDKs](/program/apis/) to code a module that does either of the following:
```

The following is an example configuration for a base modular resource implementation.
The configuration adds `acme:demo:mybase` as a modular resource from the module `my_base`.
The custom model is configured as a component with the name "my-custom-base-1".

> Commands can be sent to the base according to the Viam [base API](/components/base/#api):

Reviewer: ```suggestion
You can send commands to the base according to the Viam [base API](/components/base/#api):
```
